rules/typescript.js

```diff
@@ -146,19 +147,4 @@ module.exports = {
     // You should use type assertion style "as" instead of non-null assertion style.
     '@typescript-eslint/non-nullable-type-assertion-style': ['off'],
   },
-
-  overrides: [
-    {
-      files: ['*.@(js|cjs|mjs)'],
-      rules: {
-        // Disallow `@ts-<directive>` comments or require descriptions after directives.
-        // https://typescript-eslint.io/rules/ban-ts-comment/
-        '@typescript-eslint/ban-ts-comment': ['off'],
-
-        // Disallow `require` statements except in import statements.
-        // https://typescript-eslint.io/rules/ban-ts-comment/
-        '@typescript-eslint/no-var-requires': ['off'],
-      },
-    },
-  ],
```

Comment:
These settings are no longer necessary since this ruleset only applies to TypeScript files.

see: https://github.com/moneyforward/eslint-config-moneyforward/pull/242/files#diff-7590bcab31b10f296dc8b6d1174d6b6098b5f928b85e9f759ac96fa25a4954c0R4